src/screens/AboutScreen.tsx

```diff
@@ -1,15 +1,267 @@
+
+const profileImage: ImageSourcePropType = require("../assets/img/DummySpeakerProfilePicture.jpeg");
+
+//Temporary function to make dummy data for the organizing team
+const makeDummyData = (n = 17) => {
+	let data = [];
+	for (let i = 1; i <= n; i++) {
+		data.push({
+			id: i.toString(),
+			name: `Member ${i}`,
+			title: `Title ${i}`,
+			image: profileImage,
+		});
+	}
+	return data;
 };
 
-export default AboutScreen;
\ No newline at end of file
+const MOCK_DATA_ORGANIZING_TEAM = makeDummyData();
+
+const AboutScreen = ({
+	navigation,
+}: NativeStackScreenProps<ParamListBase, screen_names.ABOUT, undefined>) => {
+	return (
+		<SafeAreaView style={[styles.container, styles.paddingVertical]}>
+			<StatusBar
+				backgroundColor={colors.DROIDCONKE_WHITE}
+				barStyle="dark-content"
+			/>
+			<View
+				style={[
+					styles.header,
+					styles.marginVerticalSeparator,
+					styles.paddingHorizontal,
+				]}
+			>
+				<DroidconKeIcon width={150} style={styles.droidconkeIcon} />
+				<View
+					style={[
+						layoutProperties.flexRow,
+						layoutProperties.justifyBetween,
+						layoutProperties.itemsCenter,
+					]}
+				>
+					<TouchableOpacity style={styles.buttonFeedback}>
+						<Image
+							resizeMode="contain"
+							source={require("../assets/icons/SmileyIcon.png")}
+							style={styles.buttonFeedbackContentMargin}
+						/>
+						<Text
+							style={[
+								styles.buttonFeedbackText,
+								styles.buttonFeedbackContentMargin,
+							]}
+						>
+							Feedback
+						</Text>
+						<Image
+							resizeMode="contain"
+							source={require("../assets/icons/SendIcon.png")}
+						/>
+					</TouchableOpacity>
+					<TouchableOpacity style={styles.iconWrapper}>
+						<Image
+							resizeMode="contain"
+							source={require("../assets/img/profilepicture.png")}
+						/>
+					</TouchableOpacity>
+				</View>
+			</View>
+			<ScrollView showsVerticalScrollIndicator={false}>
+				<Image
+					source={require("../assets/img/about.png")}
+					resizeMode="contain"
+					style={styles.heroImage}
+				/>
+				<View style={[styles.paddingHorizontal, styles.paddingVertical]}>
+					<Text style={[styles.headingText, { marginBottom: 10 }]}>About</Text>
+					<Text style={[styles.descriptionText, { marginBottom: 28 }]}>
+						{introText}
+					</Text>
+					<Text style={[styles.headingText]}>Organizing Team</Text>
+					<View style={[styles.teamContainer]}>
+						{MOCK_DATA_ORGANIZING_TEAM.map((member) => (
+							<TeamMemberCard
+								name={member.name}
+								title={member.title}
+								profileImage={member.image}
+								onPress={() =>
+									navigation.navigate(screen_names.SINGLESPEAKER, {
+										title: "Team",
+									})
+								}
+								key={member.id}
+							/>
+						))}
+					</View>
```

Comment:
Uhm what if you used a  flatlist instead?

Reply:
So I have noticed some inconsistencies in the margin spacing as shown in this image.

![photo_2022-10-10_10-58-41](https://user-images.githubusercontent.com/28436092/194821275-7b58b489-09c7-4366-9f33-2bf8f8e859c6.jpg)

Could you check this out?

Reply:
Hi Brian, I tried the `FlatList` first, but it raised an error about nesting of virtualized list since the page content is wrapped with a `ScrollView`.   
My alternatives were: 
1. Use a `Flatlist` and place the content above the list inside the `ListHeaderComponent` and that below it inside `ListFooterComponent`. 
2. Use the map method for the team members list. 

My opinion is that  alternative 1 is *hacky* and would make the code unreadable, difficult to follow, and introduce unnecessary complexity. 

I resorted to using the map method, since the team members are not many such as to cause a performance overhead

Reply:
Hehe, I hate that virtualized list error 😆.
Okay, I get you!